evap/grades/views.py

```diff
@@ -71,6 +73,7 @@ def course_view(request, semester_id, course_id):
         grade_documents=course.grade_documents.all(),
         disable_if_archived="disabled" if semester.grade_documents_are_deleted else "",
         disable_breadcrumb_course=True,
+        infotext=Infotext.objects.get(linked_page=Infotext.LinkedPage.GRADES_PAGES),
```

Comment:
it is sufficient to place the infobox on the index page and the semester page. please remove it here.